resolvers/patient.js

```diff
@@ -1,88 +1,99 @@
-'use strict';
+"use strict";
 
-const db = require('../models');
+const db = require("../models");
 
 const patientResolvers = {
-    Query: {
-        patients: () => db.patient.findAll(),
-        patient(obj, args, context, info) {
-            return db.patient.findByPk(args.id);
-        },
+  Query: {
+    patients: () => db.patient.findAll(),
+    patient(obj, args, context, info) {
+      return db.patient.findByPk(args.id);
     },
-    Patient: {
-        collectionPointId: (obj, args, context, info) => db.collectionPoint.findByPk(obj.collectionPointId),
-        hospitalId: (obj, args, context, info) => db.hospital.findByPk(obj.hospitalId)
+    patientsByCCPId: (obj, args, context, info) =>
```

Comment:
same deal, also i'm a fan of true pascal case
```suggestion
    patientsByCcp: (obj, args, context, info) =>
```